styleguide/derek/examples/swatches.ejs

```diff
@@ -0,0 +1,214 @@
+          </div>
+        </div>
+        <div class="swatch-footer">
+          <h6>$magento-orange</h6>
+          <h6>#ed672f</h6>
+        </div>
+      </div>
+      <div class="col-sm-3 col-xs-6">
+        <div class="swatch-color purple">
+          <div class="swatch-copy-button">
+              <input id="swatch-copy" class="swatch-copy" value="5aaa28" type="text"/>
+              <a data-copytarget="#swatch-copy">Copy</a>
+          </div>
+        </div>
+        <div class="swatch-footer">
+          <h6>$purple</h6>
+          <h6>#784b91</h6>
+        </div>
+      </div>
+      <div class="col-sm-3 col-xs-6">
+        <div class="swatch-color bg-steel-blue">
+          <div class="swatch-copy-button">
+              <input id="swatch-copy" class="swatch-copy" value="5aaa28" type="text"/>
+              <a data-copytarget="#swatch-copy">Copy</a>
+          </div>
+        </div>
+        <div class="swatch-footer">
+          <h6>$steel-blue</h6>
+          <h6>#007295</h6>
+        </div>
+      </div>
+    </div>
+    <div class="row half-padding-top">
+      <div class="col-md-12">
+        <h4>Grays</h4>
+      </div>
+    </div>
+    <div class="row row-eq-height">
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color bg-light-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-lightest</h6>
+          <h6>#efefef</h6>
+        </div>
+      </div>
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color light-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-lighter</h6>
+          <h6>#dedede</h6>
+        </div>
+      </div>
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color medium-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-base</h6>
+          <h6>#666</h6>
+        </div>
+      </div>
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color medium-dark-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-darker</h6>
+          <h6>#4c4c4c</h6>
+        </div>
+      </div>
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color bg-dark-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-dark</h6>
+          <h6>#333</h6>
+        </div>
+      </div>
+      <div class="col-sm-2 col-xs-6">
+        <div class="swatch-color darkest-gray">
+        </div>
+        <div class="swatch-footer">
+          <h6>$gray-darkest</h6>
+          <h6>#2a2a2a</h6>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="container">
+    <div class="row standard-padding">
+      <div class="col-md-12">
+        <h4>Available River Colors</h4>
+        <%- partial('../_partials/solutions/rivers.ejs') %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!--thinking there should be a different js file that doesnt get included into www for this sort of js.-->
```

Comment:
@bighappyface We were discussing this.  Should this just get placed inside a `local.js` file or something that is only used internally, because tacking it into `zoolander.js` seems like unnecessary bloat if and when we ever start pulling `zoolander.js` directly.